use-timescale/services/index.md

```diff
@@ -15,6 +15,12 @@ import CloudIntro from "versionContent/_partials/_cloud-intro.mdx";
 
 <CloudIntro />
 
+## Timescale Terraform provider
+
+The [Timescale Terraform provider] [terraform-provider] provides configuration
+management resources for Timescale. You can use it to create, rename, resize,
+delete, and import service.
+
```

Comment:
Just to make the link more obvious, what if we add a second sentence like "For more information about how to get started, see the [Timescale Terraform Provider documentation site][link]." (rephrase as you like)